test/podFailover/workload/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mcr.microsoft.com/aks/fundamental/base-ubuntu:v0.0.5
+RUN apt-get update && apt-get install -y util-linux e2fsprogs mount ca-certificates udev xfsprogs
+LABEL maintainers="andyzhangx"
```

Comment:
We can omit the `maintainers` label. It's just metadata that's visible through DockerHub.